pino.js

```diff
@@ -170,7 +184,7 @@ function Pino (opts, stream) {
 }
 
 Pino.prototype = new EventEmitter()
-defineLevelsProperty(Pino.prototype)
+defineLevelsProperty(Pino.prototype, defaultLevelsValues)
```

Comment:
Tough call, we need numbers to see the effects (prototype lookups also have cost -  but then again v8 is good at amortising)